include/triton/Conversion/TritonGPUToLLVM/PtxAsmFormat.h

```diff
@@ -104,6 +107,31 @@ struct PTXBuilder {
   // Create a list of operands.
   Operand *newListOperand() { return newOperand(); }
 
+  Operand *newListOperand(ArrayRef<std::pair<mlir::Value, std::string>> items) {
+    auto *list = newOperand();
+    for (auto &item : items) {
+      list->listAppend(newOperand(item.first, item.second));
+    }
+    return list;
+  }
+
+  Operand *newListOperand(unsigned count, mlir::Value val,
+                          const std::string &constraint) {
+    auto *list = newOperand();
+    for (int i = 0; i < count; i++) {
```

Comment:
Nit: `unsigned i` and `++i` :)

Reply:
Yeah, I've rewritten all the forloop `x++`  to `++x`.